src/controllers/lyrics.js

```diff
@@ -17,6 +19,7 @@ let currentItem;
 
 let savedLyrics;
 let isDynamicLyric = false;
+let autoScroll = true;
```

Comment:
I think we should set `autoScroll` to `true` in the `viewshow` handler so that we autoscroll when switching between views.

Reply:
Whoops. Forgot to consider that scenario.